{"errorMessage":"Lambda process returned error status code: 1. Result: {\"errorType\":\"Runtime.HandlerNotFound\",\"errorMessage\":\"index.handler is undefined or not exported\"}. Output:\n2022-03-16T20:06:21.424Z\tundefined\tERROR\tUncaught Exception\t{\"errorType\":\"Runtime.HandlerNotFound\",\"errorMessage\":\"index.handler is undefined or not exported\",\"stack\":[\"Runtime.HandlerNotFound: index.handler is undefined or not exported\",\"    at Object.module.exports.load (/var/runtime/UserFunction.js:144:11)\",\"    at Object.<anonymous> (/var/runtime/index.js:45:30)\",\"    at Module._compile (internal/modules/cjs/loader.js:778:30)\",\"    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)\",\"    at Module.load (internal/modules/cjs/loader.js:653:32)\",\"    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\",\"    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\",\"    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)\",\"    at startup (internal/bootstrap/node.js:283:19)\",\"    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)\"]}\n\u001b[32mSTART RequestId: 7b6c8a65-ce17-1f23-ef83-83663fef3347 Version: $LATEST\u001b[0m\n\u001b[32mEND RequestId: 7b6c8a65-ce17-1f23-ef83-83663fef3347\u001b[0m\n\u001b[32mREPORT RequestId: 7b6c8a65-ce17-1f23-ef83-83663fef3347\tInit Duration: 159.94 ms\tDuration: 2.89 ms\tBilled Duration: 3 ms\tMemory Size: 1536 MB\tMax Memory Used: 44 MB\t\u001b[0m","errorType":"InvocationException","stackTrace":["  File \"/opt/code/localstack/localstack/services/awslambda/lambda_api.py\", line 580, in run_lambda\n    result = LAMBDA_EXECUTOR.execute(func_arn, func_details, event, context=context,\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 176, in execute\n    return do_execute()\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 168, in do_execute\n    return _run(func_arn=func_arn)\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 149, in wrapped\n    raise e\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 145, in wrapped\n    result = func(*args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 159, in _run\n    raise e\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 147, in _run\n    result = self._execute(func_arn, func_details, event, context, version)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 325, in _execute\n    result = self.run_lambda_executor(cmd, stdin, env_vars=environment, func_details=func_details)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 231, in run_lambda_executor\n    raise InvocationException('Lambda process returned error status code: %s. Result: %s. Output:\\n%s' %\n"]}
